# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1322363886 18000
#      Sat Nov 26 22:18:06 2011 -0500
# Node ID 8ebff542d1330ffe603fec4941c553cfe6503cd3
# Parent  c55bd4a7a3e41ff82082dc65d6e666bd0ad7469e
doc: Remove obsolete "keylabel" and replace by "displayname".

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1803,32 +1803,28 @@ The @code{line} properties are:
 @item color
 The RGB color of the line, or a color name.  @xref{Colors}.
 
 @item createfcn
 
 @item deletefcn
 
 @item displayname
+The text of the legend entry corresponding to this line.
 
 @item erasemode
 
 @item handlevisibility
 
 @item hittest
 
 @item interpreter
 
 @item interruptible
 
-@item keylabel
-The text of the legend entry corresponding to this line.  Note that this
-property is not compatible with @sc{matlab} and may be removed in a
-future version of Octave.
-
 @item ldata
 The lower errorbar in the y direction to be plotted.
 
 @item linestyle
 @itemx linewidth
 @xref{Line Styles}.
 
 @item linewidth
@@ -1920,16 +1916,17 @@ The @code{text} properties are:
 @item color
 The color of the text.  @xref{Colors}.
 
 @item createfcn
 
 @item deletefcn
 
 @item displayname
+The text of the legend entry corresponding to this line.
 
 @item edgecolor
 
 @item editing
 
 @item erasemode
 
 @item fontangle
@@ -2127,16 +2124,19 @@ Data defining the patch object.
 @item clipping
 
 @item createfcn
 
 @item deletefcn
 
 @item diffusestrength
 
+@item displayname
+The text of the legend entry corresponding to this line.
+
 @item edgealpha
 
 @item edgecolor
 The color of the line defining the patch.  @xref{Colors}.
 
 @item edgelighting
 
 @item erasemode
@@ -2158,18 +2158,16 @@ The fill color of the patch.  @xref{Colo
 @item handlevisibility
 
 @item hittest
 
 @item interpreter
 
 @item interruptible
 
-@item keylabel
-
 @item linestyle
 @xref{Line Styles}.
 
 @item linewidth
 @xref{Line Styles}.
 
 @item marker
 @xref{Marker Styles}.
@@ -2277,16 +2275,19 @@ The @code{surface} properties are:
 @item clipping
 
 @item createfcn
 
 @item deletefcn
 
 @item diffusestrength
 
+@item displayname
+The text of the legend entry corresponding to this surface.
+
 @item edgealpha
 
 @item edgecolor
 
 @item edgelighting
 
 @item erasemode
 
@@ -2299,21 +2300,16 @@ The @code{surface} properties are:
 @item handlevisibility
 
 @item hittest
 
 @item interpreter
 
 @item interruptible
 
-@item keylabel
-The text of the legend entry corresponding to this surface.  Note that
-this property is not compatible with @sc{matlab} and may be removed in a
-future version of Octave.
-
 @item linestyle
 
 @item linewidth
 
 @item marker
 
 @item markeredgecolor
 
